<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_CamGenWidget</class>
 <widget class="QWidget" name="C_CamGenWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_CamGenWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgeWiSectionHeader" name="pc_WiTitle" native="true"/>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_CamOgeSpi" name="pc_Splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="stw::opensyde_gui::C_CamGenMessagesWidget" name="pc_WidgetMessages" native="true"/>
     <widget class="stw::opensyde_gui::C_CamGenSigWidget" name="pc_WidgetSignals" native="true"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgeSpi</class>
   <extends>QSplitter</extends>
   <header>C_CamOgeSpi.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_CamOgeWiSectionHeader</class>
   <extends>QWidget</extends>
   <header>C_CamOgeWiSectionHeader.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_CamGenMessagesWidget</class>
   <extends>QWidget</extends>
   <header>C_CamGenMessagesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_CamGenSigWidget</class>
   <extends>QWidget</extends>
   <header>C_CamGenSigWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
